### PR TITLE
added days supplied to medication

### PR DESCRIPTION
**Pull request by daco101**
This PR adds a "Days Supplied" attribute to: Medication Order, Dispensed and Discharge. The field is a number indicating the number of days a supply of medication should last.

Pull requests into Health Data Standards require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.
 
**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1690
- [x] Internal ticket links back to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] Code coverage has not gone down and all code touched or added is covered.
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here:
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here:
 
**Cypress Reviewer:**
 
Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code
 
**Bonnie Reviewer:**
 
Name: @hossenlopp
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: test/unit/models/medication_test.rb [hunk 1]
```diff
@@ -9,4 +10,8 @@ def setup
   def test_setting_not_present
     assert @medication.respond_to?(:setting)
   end
+  # setting attribute added to medication order in QDM 5.4
+  def test_days_supplied_not_present
```
Comment (mayerm94) [inline test/unit/models/medication_test.rb]: should be test_days_supplied_present, the setting test name above is also wrong and has been corrected in the BONNIE_1691 branch

Comment (hossenlopp) [reply test/unit/models/medication_test.rb]: Should that change be made here?

**Review by hossenlopp — APPROVED** [review]

**Review by mayerm94 — APPROVED** [review]
